README.md

```diff
@@ -87,6 +95,26 @@ ENV Variable | Description |
 `TEST_CLASSROOM_OWNER_ORGANIZATION_GITHUB_ID` | GitHub ID (preferably one created specifically for testing against).
 `TEST_CLASSROOM_OWNER_ORGANIZATION_GITHUB_LOGIN` | GitHub login (preferably one created specifically for testing against).
 
+To obtain these values you will need:
+
+* A teacher (your primary GitHub account)
+* A student (another Github account, created for this purpose)
+* An organisation that the teacher is an owner of, but the student does not belong to.
+
+It is best if you create your own organization for testing purposes, if you have done so:
+
+To obtain the `OWNER_GITHUB_ID' value, you can go to `https://api.github.com/orgs/organization_owner_username`.
```

Comment:
```diff
+ To obtain the `OWNER_GITHUB_ID` value, you can go to `https://api.github.com/orgs/organization_owner_username`.
- To obtain the `OWNER_GITHUB_ID' value, you can go to `https://api.github.com/orgs/organization_owner_username`.
```

You used a `'` instead of a ` :smiley: